mlir/lib/Dialect/Rock/Transforms/FoldBroadcast.cpp

```diff
@@ -202,6 +202,10 @@ struct FoldBroadcast : public OpRewritePattern<rock::GemmOp> {
         op.getNumCUAttr(), op.getFeatures(), op.getStoreMethod(),
         op.getDerivedBlockSizeAttr(), op.getGridSizeAttr(), op.getParamsAttr());
 
+    // Convert optional attributes
+    if (auto attr = (*op).template getAttrOfType<StringAttr>("perf_config"))
+      gemm->setAttr("perf_config", attr);
```

Comment:
Can you attach perfConfig in one of the tests inside `fold_broadcast.mlir` and check that it exists after running the pass ?